data/presets/presets/man_made/observatory.json

```diff
@@ -0,0 +1,15 @@
+{
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "type": [
+        "astronomical",
+        "meteorological",
+        "espionage"
```

Comment:
I'd remove this word "espionage", it's not really a term related to an [observatory](https://en.wikipedia.org/wiki/Observatory).